### DISCUSSION TITLE
Hubs Enterprise Multi-Server

### DISCUSSION THREAD

**Discussion by Vhive2012** [mozilla/hubs#4864] [Hubs Cloud Questions] [answered]
Hi,

I selected the Hubs Enterprise Multi-Server in AWS Marketplace and when I went to create the stack in Cloud Formation it didn't give me the option to create a separate App & Voice server i.e. 1x1?

I've been able to previously so not sure if something has changed on version 1.1.2? 

When the stack created I went into EC2 and only saw one App Server and no Voice/Stream server?

This is the stack template I used..

https://s3.amazonaws.com/awsmp-fulfillment-cf-templates-prod/94ec4ef1-285d-4e63-b2eb-0b65ea93b942.a2018a6b-4c1f-4381-ba9a-47819e2c6dd4.template

Please advise?

Thanks,

Dean.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/HUBS-1170)

**Comment by vvdt** [comment]
I had exactly the same question. I also missed these fields when trying to upgrade our exisiting Hubs enterprise multi-server environment on AWS. 

If this is intentional, is there any information on how to set the number of servers in the new situation?

**Comment by Vhive2012** [comment]
Do you know when this will be resolved?

It defeats the purpose of using Hubs Enterprise if you are unable to use a
multi-server setup.

**Reply by pattersonbl2** [reply]
We are working on the issues this week we don't have a eta on completion

**Comment by Vhive2012** [comment, marked as answer]
Ok, thanks for the update.

**Reply by vvdt** [reply]
Hi, @Pattersonbl2  any luck you can give an update on this issue?

**Reply by pattersonbl2** [reply]
Missed this thread but it's been fixed since dec 2nd

**Comment by vvdt** [comment]
Thanks 🙏
